packages/types/src/exchange/params.ts

```diff
@@ -0,0 +1,29 @@
+import {ExchangeBlockchainCode} from './blockchain'
+import {ExchangeCoin} from './coin'
+import {ExchangeFiat} from './fiat'
+
+export type ExchangeReferralUrlQueryStringParams = {
+  /** Can be used to indicate whether the customer wants to sell by passing through a value of sell. The customer will be shown the Sell Order Form when they are redirected to teh Exchange. If this parameter is not passed, then the order will default to buy */
+  orderType?: 'sell' | 'buy'
```

Comment:
I think it must be required. (banxa will fallback to sell) but at manager level it always requires